NotadogApi.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Xunit;
+
+namespace NotadogApi.Tests
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
+        {
+            var result = true;
+            Assert.True(result, "result should not be true");

```

Comment:
The second param is needed for assertion fail. So it is better "result is not true" or "result should be true".